app/services/reinstate_declined_offer.rb

```diff
@@ -0,0 +1,33 @@
+class ReinstateDeclinedOffer
+  def initialize(course_choice:, zendesk_ticket:)
+    @course_choice = course_choice
+    @zendesk_ticket = zendesk_ticket
+  end
+
+  def save!
+    reset_all_dbd
+
+    @course_choice.update!(
+      status: 'offer',
+      declined_at: nil,
+      declined_by_default: false,
+      decline_by_default_at: 10.business_days.from_now,
```

Comment:
See below

Reply:
"below" in the file, [above](https://github.com/DFE-Digital/apply-for-teacher-training/pull/4572#discussion_r620313500) in the series of review comments!